pkg/events/events.go

```diff
@@ -123,6 +123,10 @@ func New(object interface{}, eventType config.EventType, resource, clusterName s
 		event.Count = eventObj.Count
 		event.Action = eventObj.Action
 		event.TimeStamp = eventObj.LastTimestamp.Time
+		t := time.Time{}
+		if eventObj.LastTimestamp.Time == t {
```

Comment:
@Lion916 lets use `time.IsZero()` method to check nil time
https://pkg.go.dev/time#Time.IsZero

Reply:
@PrasadG193    Could you help to review another PR? Thanks~   https://github.com/infracloudio/botkube/pull/572

Reply:
@Lion916 could you please make the changes suggested above? :arrow_up: